server/src/test/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceReconcilerTests.java

```diff
@@ -1238,7 +1238,7 @@ public void testRebalanceDoesNotCauseHotSpots() {
             assertThat(
                 "Reconciling nodes should all have same amount (max 1 delta) of moves: " + totalOutgoingMoves,
                 summary.getMax() - summary.getMin(),
-                lessThanOrEqualTo(1)
+                lessThanOrEqualTo(numberOfNodes)
```

Comment:
number of concurrent rebalance now is controlled by `cluster.routing.allocation.node_concurrent_incoming_recoveries / cluster.routing.allocation.node_concurrent_outgoing_recoveries`

which has default value  = number of nodes.